### ISSUE TITLE
It is possible to add a new feature to read zip parts?

### ISSUE THREAD

**Issue by radebla** [nktknshn/tgmount-ng#12]
Mounted zip files can open as folders it is great! but I want to read or mount .z00 .z01 .z02 parts of same zipped file.

I'm diving about ZipFile code but I don't understand how id message works on this code to delete it or modify it becasue 123_example_file.zip
124_example_file.z00
125_example_file-z01
Can't read zip parts because file name is different...

Kind Regards.

**Comment by nktknshn** [comment]
It seems possible to mount such an archive. The only challenge I see so far is automatically detecting a multipart archive from the filenames, as they are not very conventional.

**Comment by radebla** [comment]
That's correct, for these case it is interesting delete filename prefix. You answered me this question at other issue. Thank you.

https://github.com/nktknshn/tgmount-ng/issues/11#issuecomment-1712513169
